modules/highlight/highlight.go

```diff
@@ -82,6 +87,11 @@ func NewContext() {
 // FileNameToHighlightClass returns the best match for highlight class name
 // based on the rule of highlight.js.
 func FileNameToHighlightClass(fname string) string {
+
+	if name, ok := highlightSpecificNames[fname]; ok {
```

Comment:
Maybe we should move these after ignore check.

Reply:
I don't think that it does change anything as they are exclusively different maps

Reply:
Review it I ask my self if we shouldn't replace highlightFileNames with map[string]string directly and only one check.

Reply:
Yes, seems like this is duplicating with the purpose `highlightFileNames`, so I'd say lowercase the name and put it there instead.

Reply:
At least cmake requires CMakeFiles.txt file to be exactly like this named, so it is case-sensitive

Reply:
I think this may also be true for `Dockerfile` and `Makefile` actually, for which we currently do that case-insensitve check, so I guess those could be moved to sensitive checks too.

Reply:
I think that `make` [command](https://www.gnu.org/software/make/manual/make.html#Makefile-Names) works also with `GNUmakefile` and `makefile` filenames, whereas `cmake` only works with `CMakeFiles.txt`. `GNUmakefile` could be put in sensitive checks too.

Reply:
> By default, when make looks for the makefile, it tries the following names, in order: GNUmakefile, makefile and Makefile

We can lower-case all. I don't think it would be too bad to highlight cmakefiles.txt like CMakeFiles.txt. Futhermore, I don't know how it work with case-insensitive fs.